rest/src/main/java/fi/otavanopisto/pyramus/rest/ObjectFactory.java

```diff
@@ -681,7 +686,17 @@ public Object map(StudentStudyEndReason entity) {
           @Override
           public Object map(StudentContactLogEntry entity) {
             StudentContactLogEntryType type = StudentContactLogEntryType.valueOf(entity.getType().name());
-            return new fi.otavanopisto.pyramus.rest.model.StudentContactLogEntry(entity.getId(), entity.getText(), entity.getCreatorName(), toOffsetDateTime(entity.getEntryDate()), type, entity.getArchived());
+            
+            Object comments = createModel(studentContactLogEntryCommentController.listContactLogEntryCommentsByEntry(entity));
```

Comment:
Should be safe to cast the return value of `createModel` as `List<StudentContactLogEntryCommentRestModel>`